Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM node:10-alpine
+FROM node:16-alpine
 
 COPY . /usr/app
 WORKDIR /usr/app
-RUN npm install
+RUN npm ci 
```

Comment:
maybe I'm missing something, but `ci` doesn't seem to be a script target in the package.json ?